piplicenses.py

```diff
@@ -1169,7 +1169,8 @@ def create_parser(
         action="store_true",
         default=config_from_file.get("with-license-file", False),
         help="dump with location of license file and "
-        "contents, most useful with JSON output",
+        "contents, most useful with JSON output. "
+        "For structured formats (CSV, Markdown, reST), see README for workflow examples.",
```

Comment:
:broom: NIT - Linter Complains about line length.

:wrench: see [Logs](https://github.com/raimon49/pip-licenses/actions/runs/18637329541/job/53220547434?pr=244#step:6:100)


Consider splitting this line with additional hard folding:

```suggestion
        "For structured formats (CSV, Markdown, reST), "
        "see README for workflow examples.",
```